tests/testdata/codegen/dashed-import-schema/docs/tree/v1/rubbertree/_index.md

```diff
@@ -501,7 +501,7 @@ The RubberTree resource accepts the following [input](/docs/intro/concepts/input
 <a data-swiftype-name="resource-property" data-swiftype-type="text" href="#container_python" style="color: inherit; text-decoration: inherit;">container</a>
 </span>
         <span class="property-indicator"></span>
-        <span class="property-type"><a href="#container">Container<wbr>Args</a></span>
+        <span class="property-type"><a href="#container">root_<wbr>inputs.<wbr>Container<wbr>Args</a></span>
```

Comment:
This doesn't make sense to expose. The "root" thing is also an internal implementation detail. What this is really saying is you'd have something like:

```python
import pulumi_plant

# usage
pulumi_plant.ContainerArgs
```

Or

```python
from pulumi_plant import ContainerArgs

# usage
ContainerArgs
```

I think we'd want this to be `pulumi_plant.ContainerArgs` or possibly better just `ContainerArgs` since it is top-level and we seem to be showing top-level submodules without prefixing the top-level package module name.

Reply:
Yes, that's a great point. We aren't consistent about displaying this in other languages: C# shows it with the package module name, while Node.js doesn't. I'll take a stab at improving this, thank you.